source/includes/api/_before_you_get_started.md

```diff
@@ -56,6 +56,31 @@ Below you can find a list of RW API resources that use the `application` field:
 * [Vocabulary](/index-rw.html#vocabulary-and-tags)
 * [Widgets](/index-rw.html#widget10)
 
+## Caching
+
+Some services in the RW API rely on caching as a way to improve the performance and efficiency of their endpoints. By relying on caching mechanisms, services can serve pre-computed responses for commonly requested endpoints, reducing the need to perform the computations needed to serve the same request several times. Caching also has the added benefits of improving performance and reducing service load, which leads to better scalability overall.
+
+The RW API has a system-wide HTTP cache that you may use to cache your requests. This cache is based on [Fastly](https://www.fastly.com/), so please refer to its documentation if you are looking for a specific detail on its behavior. HTTP caching is opt-in, meaning that by default no caching mechanism is applied to any request -  if you'd like your endpoints to benefit from caching, you need to explicitly implement it.
```

Comment:
the fact that we use fastly is an implementation detail, and we shouldn't really mention it here. Perhaps it would make sense on a dev docs section, but here it's not relevant IMO. Instead, what we can do is briefly explain how an HTTP cache works: when a request comes in, it computes the response, saves it, and subsequent requests are then replied with the save response, without the need for recomputing it. I doubt anyone reading this does not know how a cache works, so it may be redundant, but I feel like that's more important background info for an API user to have.

Reply:
Agreed - we have cache implementation details on the developer docs, so I removed redundant information from here and added a reference to the developer docs :+1: